pkg/config/templates/rke2-92-harvester-kube-audit-policy.yaml

```diff
@@ -1,4 +1,14 @@
 apiVersion: audit.k8s.io/v1
 kind: Policy
+omitStages:
+  - "ResponseStarted"
+  - "ResponseComplete"
 rules:
-- level: Metadata
+  # Any include/exclude rules are added here
+
+  # A catch-all rule to log all other (create/delete/patch) requests at the Metadata level
+  - level: Metadata
+    verbs: ["create", "delete", "patch"]
```

Comment:
just wondering if it would be better for us to watch desired resources only like VM, PVC, image, and system-level resources like pod or configmap, secrets might be less important for now.

Reply:
It needs more time to investigate. 

I tried to filter out specific resources (e.g. `rolebinding`), but failed. When conditions are conflict/not meeting requirements,  `apiserver` simply records nothing, no clear clues are left. Need to check `kube-apiserver`'s code to understand more details about audit-policy.

When `white-list` is adopt, we may need to update this file continuously, more resources are required to be recorded for better audit. 

Anyway, it's relatively flexible to update the context of this file later after it's well verified.

Reply:
sure, we can leave it for either v1.1.1 or rc2 upon the time, should be fine to keep the existing one for now, thanks.